src/native/managed/cdacreader/src/Contracts/Loader_1.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Diagnostics.DataContractReader.Contracts;
+
+internal readonly struct Loader_1 : ILoader
+{
+    private readonly Target _target;
+
+    internal Loader_1(Target target)
+    {
+        _target = target;
+    }
+
+    ModuleHandle ILoader.GetModuleHandle(TargetPointer modulePointer)
+    {
+        return new ModuleHandle(modulePointer);
```

Comment:
should we do some kind of validation here. at least check for `modulePointer == TargetPointer.Null` ?